sslbl/package.yaml

```diff
@@ -0,0 +1,61 @@
+id: sslbl
+name: SSLBL
+author: Tenzir
+author_icon: https://github.com/tenzir.png
+package_icon: |
+  https://raw.githubusercontent.com/tenzir/library/main/sslbl/package.svg
+description: |
+  The [SSLBL](https://sslbl.abuse.ch/) package makes available a lookup table
+  with SHA1 hashes of blacklisted certificates that can be used when monitoring
+  SSL/TLS certificate exchanges.
+
+inputs:
+  refresh-interval:
+    name: Time between context updates
+    description: |
+      The duration between updates that fetch the SSLBL database via the API.
+    default: 1 hour
+
+contexts:
+  feodo:
```

Comment:
Wrong context name—did you test this package locally?

Reply:
Yes, but I ended up here: https://github.com/tenzir/tenzir/pull/4568#discussion_r1749166210